test/uk/gov/hmrc/economiccrimelevyregistration/navigation/UtrPageNavigatorSpec.scala

```diff
@@ -52,7 +53,8 @@ class UtrPageDeregisterNavigatorSpec extends SpecBase {
       val otherEntityJourneyData = OtherEntityJourneyData
         .empty()
         .copy(
-          ctUtr = Some(utr)
+          ctUtr = Some(utr),
+          companyRegistrationNumber = Some(random[String])
```

Comment:
See other comments. You don't need random

Reply:
Changed.